packages/bundler-plugin-core/src/types.ts

```diff
@@ -241,9 +241,25 @@ export interface Options {
      * Defaults to `false`.
      */
     injectBuildInformation?: boolean;
+
+    /**
+     * Metadata associated with this module.
+     * The metadata is serialized and can be looked up at runtime by URL.
+     *
+     * Note: This option is currently only supported by `@sentry/webpack-plugin`.
+     */
+    moduleMetadata?: object | ModuleMetadataCallback;
   };
 }
 
+export interface ModuleMetadataCallbackArgs {
+  org?: string;
+  project?: string;
+  release?: string;
+}
+
+export type ModuleMetadataCallback = (args: ModuleMetadataCallbackArgs) => object;
```

Comment:
Should we set the return type of this function to `any`? Users are allowed to return anything they want here in theory.

Reply:
Yeah I think that's probably a good idea